[2024-07-03T09:57:03.399+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: reports_workflow.id_task_get_emails scheduled__2024-07-02T05:00:00+00:00 [queued]>
[2024-07-03T09:57:03.416+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: reports_workflow.id_task_get_emails scheduled__2024-07-02T05:00:00+00:00 [queued]>
[2024-07-03T09:57:03.416+0000] {taskinstance.py:1361} INFO - Starting attempt 1 of 1
[2024-07-03T09:57:03.435+0000] {taskinstance.py:1382} INFO - Executing <Task(PythonOperator): id_task_get_emails> on 2024-07-02 05:00:00+00:00
[2024-07-03T09:57:03.441+0000] {standard_task_runner.py:57} INFO - Started process 20442 to run task
[2024-07-03T09:57:03.450+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'reports_workflow', 'id_task_get_emails', 'scheduled__2024-07-02T05:00:00+00:00', '--job-id', '303', '--raw', '--subdir', 'DAGS_FOLDER/dag_email_robot.py', '--cfg-path', '/tmp/tmpdy5n6ni0']
[2024-07-03T09:57:03.451+0000] {standard_task_runner.py:85} INFO - Job 303: Subtask id_task_get_emails
[2024-07-03T09:57:03.528+0000] {task_command.py:415} INFO - Running <TaskInstance: reports_workflow.id_task_get_emails scheduled__2024-07-02T05:00:00+00:00 [running]> on host 6f4ef5488251
[2024-07-03T09:57:03.679+0000] {taskinstance.py:1660} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='FIRDAVS' AIRFLOW_CTX_DAG_ID='reports_workflow' AIRFLOW_CTX_TASK_ID='id_task_get_emails' AIRFLOW_CTX_EXECUTION_DATE='2024-07-02T05:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-07-02T05:00:00+00:00'
[2024-07-03T09:57:03.679+0000] {logging_mixin.py:151} INFO - path of config file:  /opt/***
[2024-07-03T09:57:03.680+0000] {taskinstance.py:1943} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/operators/python.py", line 192, in execute
    return_value = self.execute_callable()
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/operators/python.py", line 209, in execute_callable
    return self.python_callable(*self.op_args, **self.op_kwargs)
  File "/opt/airflow/dags/dag_email_robot.py", line 22, in get_emails
    email_conf = json.load (open(path / ("config/email_conf.json")))
NameError: name 'json' is not defined
[2024-07-03T09:57:03.690+0000] {taskinstance.py:1400} INFO - Marking task as FAILED. dag_id=reports_workflow, task_id=id_task_get_emails, execution_date=20240702T050000, start_date=20240703T095703, end_date=20240703T095703
[2024-07-03T09:57:03.703+0000] {standard_task_runner.py:104} ERROR - Failed to execute job 303 for task id_task_get_emails (name 'json' is not defined; 20442)
[2024-07-03T09:57:03.747+0000] {local_task_job_runner.py:228} INFO - Task exited with return code 1
[2024-07-03T09:57:03.772+0000] {taskinstance.py:2784} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-07-03T12:04:40.140+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: reports_workflow.id_task_get_emails scheduled__2024-07-02T05:00:00+00:00 [queued]>
[2024-07-03T12:04:40.170+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: reports_workflow.id_task_get_emails scheduled__2024-07-02T05:00:00+00:00 [queued]>
[2024-07-03T12:04:40.170+0000] {taskinstance.py:1361} INFO - Starting attempt 1 of 1
[2024-07-03T12:04:40.190+0000] {taskinstance.py:1382} INFO - Executing <Task(PythonOperator): id_task_get_emails> on 2024-07-02 05:00:00+00:00
[2024-07-03T12:04:40.196+0000] {standard_task_runner.py:57} INFO - Started process 23264 to run task
[2024-07-03T12:04:40.217+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'reports_workflow', 'id_task_get_emails', 'scheduled__2024-07-02T05:00:00+00:00', '--job-id', '746', '--raw', '--subdir', 'DAGS_FOLDER/dag_email_robot.py', '--cfg-path', '/tmp/tmp7kzyo2np']
[2024-07-03T12:04:40.219+0000] {standard_task_runner.py:85} INFO - Job 746: Subtask id_task_get_emails
[2024-07-03T12:04:40.295+0000] {task_command.py:415} INFO - Running <TaskInstance: reports_workflow.id_task_get_emails scheduled__2024-07-02T05:00:00+00:00 [running]> on host 6f4ef5488251
[2024-07-03T12:04:40.392+0000] {taskinstance.py:1660} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='FIRDAVS' AIRFLOW_CTX_DAG_ID='reports_workflow' AIRFLOW_CTX_TASK_ID='id_task_get_emails' AIRFLOW_CTX_EXECUTION_DATE='2024-07-02T05:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-07-02T05:00:00+00:00'
[2024-07-03T12:04:40.394+0000] {logging_mixin.py:151} INFO - path of config file:  /opt/***
[2024-07-03T12:04:40.394+0000] {logging_mixin.py:151} INFO - #>2024-07-03 12:04:40.394811_User@StatDep: EmailRobot in screated.
[2024-07-03T12:04:40.395+0000] {logging_mixin.py:151} INFO - #>2024-07-03 12:04:40.395025_User@StatDep: get_for_recent_emails is started...
[2024-07-03T12:04:40.395+0000] {logging_mixin.py:151} INFO - #>2024-07-03 12:04:40.395161_User@StatDep: pop3_fetch_emails isstarted...
[2024-07-03T12:04:40.789+0000] {logging_mixin.py:151} INFO -    b'+OK'
[2024-07-03T12:04:41.219+0000] {logging_mixin.py:151} INFO -    mailbox have 1 emails and the robot can process 10
[2024-07-03T12:04:41.642+0000] {logging_mixin.py:151} INFO -    #>2024-07-03 12:04:41.642005_conneciton to db is set.
[2024-07-03T12:04:41.647+0000] {taskinstance.py:1943} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/operators/python.py", line 192, in execute
    return_value = self.execute_callable()
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/operators/python.py", line 209, in execute_callable
    return self.python_callable(*self.op_args, **self.op_kwargs)
  File "/opt/airflow/dags/dag_email_robot.py", line 25, in get_emails
    robot.fetch_emails()
  File "/opt/airflow/dags/emailRobot.py", line 62, in fetch_emails
    return self.pop3_fetch_emails()
  File "/opt/airflow/dags/emailRobot.py", line 156, in pop3_fetch_emails
    cursor.execute (query, insert_values)
psycopg2.errors.UndefinedTable: relation "sma_stat_dep.tbl_file_upload" does not exist
LINE 1: ...) AS (INSERT INTO                                 sma_stat_d...
                                                             ^

[2024-07-03T12:04:41.660+0000] {taskinstance.py:1400} INFO - Marking task as FAILED. dag_id=reports_workflow, task_id=id_task_get_emails, execution_date=20240702T050000, start_date=20240703T120440, end_date=20240703T120441
[2024-07-03T12:04:41.673+0000] {standard_task_runner.py:104} ERROR - Failed to execute job 746 for task id_task_get_emails (relation "sma_stat_dep.tbl_file_upload" does not exist
LINE 1: ...) AS (INSERT INTO                                 sma_stat_d...
                                                             ^
; 23264)
[2024-07-03T12:04:41.685+0000] {local_task_job_runner.py:228} INFO - Task exited with return code 1
[2024-07-03T12:04:41.709+0000] {taskinstance.py:2784} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-07-03T12:21:54.143+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: reports_workflow.id_task_get_emails scheduled__2024-07-02T05:00:00+00:00 [queued]>
[2024-07-03T12:21:54.165+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: reports_workflow.id_task_get_emails scheduled__2024-07-02T05:00:00+00:00 [queued]>
[2024-07-03T12:21:54.165+0000] {taskinstance.py:1361} INFO - Starting attempt 1 of 1
[2024-07-03T12:21:54.181+0000] {taskinstance.py:1382} INFO - Executing <Task(PythonOperator): id_task_get_emails> on 2024-07-02 05:00:00+00:00
[2024-07-03T12:21:54.188+0000] {standard_task_runner.py:57} INFO - Started process 24043 to run task
[2024-07-03T12:21:54.195+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'reports_workflow', 'id_task_get_emails', 'scheduled__2024-07-02T05:00:00+00:00', '--job-id', '1006', '--raw', '--subdir', 'DAGS_FOLDER/dag_email_robot.py', '--cfg-path', '/tmp/tmpl5spooec']
[2024-07-03T12:21:54.196+0000] {standard_task_runner.py:85} INFO - Job 1006: Subtask id_task_get_emails
[2024-07-03T12:21:54.256+0000] {task_command.py:415} INFO - Running <TaskInstance: reports_workflow.id_task_get_emails scheduled__2024-07-02T05:00:00+00:00 [running]> on host 6f4ef5488251
[2024-07-03T12:21:54.467+0000] {taskinstance.py:1660} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='FIRDAVS' AIRFLOW_CTX_DAG_ID='reports_workflow' AIRFLOW_CTX_TASK_ID='id_task_get_emails' AIRFLOW_CTX_EXECUTION_DATE='2024-07-02T05:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-07-02T05:00:00+00:00'
[2024-07-03T12:21:54.468+0000] {logging_mixin.py:151} INFO - path of config file:  /opt/***
[2024-07-03T12:21:54.468+0000] {logging_mixin.py:151} INFO - #>2024-07-03 12:21:54.468588_User@StatDep: EmailRobot in screated.
[2024-07-03T12:21:54.468+0000] {logging_mixin.py:151} INFO - #>2024-07-03 12:21:54.468775_User@StatDep: get_for_recent_emails is started...
[2024-07-03T12:21:54.468+0000] {logging_mixin.py:151} INFO - #>2024-07-03 12:21:54.468908_User@StatDep: pop3_fetch_emails isstarted...
[2024-07-03T12:21:54.885+0000] {logging_mixin.py:151} INFO -    b'+OK'
[2024-07-03T12:21:55.163+0000] {logging_mixin.py:151} INFO -    mailbox have 0 emails and the robot can process 10
[2024-07-03T12:21:55.163+0000] {logging_mixin.py:151} INFO - #>2024-07-03 12:21:55.163799_User@StatDep: no emails to process, exiting pop3_fetch_emails()...
[2024-07-03T12:21:55.164+0000] {python.py:194} INFO - Done. Returned value was: True
[2024-07-03T12:21:55.221+0000] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=reports_workflow, task_id=id_task_get_emails, execution_date=20240702T050000, start_date=20240703T122154, end_date=20240703T122155
[2024-07-03T12:21:55.383+0000] {local_task_job_runner.py:228} INFO - Task exited with return code 0
[2024-07-03T12:21:55.440+0000] {taskinstance.py:2784} INFO - 1 downstream tasks scheduled from follow-on schedule check
